robottelo/constants.py

```diff
@@ -962,3 +962,60 @@
     u'alpha', u'numeric', u'alphanumeric',
     u'latin1', u'utf8', u'cjk', u'html'
 ]
+
+
+# All UI crud classes listed here to allow dynamical import
+# import import_string; import_string('robottelo.ui.{0}'.format(item))
+UI_CRUD = [
```

Comment:
I am wondering if we could import this dynamically, like getting all classes available under robottelo.ui and its submodules. This way we don't need to update this list whenever a new class or new module is added.